README.rst

```diff
@@ -4,11 +4,11 @@
     :target: https://crate.io
 
 =================================
-CRATE: Put data to work. Simply.
+CRATEDB: Put data to work. Simply.
 =================================
```

Comment:
title undeline too short.
needs to have at least same length as title itself. ([rst](http://docutils.sourceforge.net/docs/user/rst/quickref.html#section-structure))

please check all occurrences!